### PR TITLE
doc: fix documented mock request example

### PR DESCRIPTION
**Pull request by austinkelleher**
Small fix to a [mock request example](https://undici.nodejs.org/#/docs/best-practices/mocking-request?id=mocking-request) that was broken.

### CODE REVIEW

File: docs/best-practices/mocking-request.md [hunk 1]
```diff
@@ -5,17 +5,20 @@ Undici have its own mocking [utility](../api/MockAgent.md). It allow us to inter
 Example:
 
 ```js
-// index.mjs
+// bank.mjs
```
Comment (austinkelleher) [inline docs/best-practices/mocking-request.md:8]: Renamed to `bank.mjs`. The `import` in the example below was incorrect by referencing `undici.mjs`. Happy to change this comment to `undici.mjs` if preferred.

File: docs/best-practices/mocking-request.md [hunk 2]
```diff
@@ -5,17 +5,20 @@ Undici have its own mocking [utility](../api/MockAgent.md). It allow us to inter
...
 import { request } from 'undici'
 
 export async function bankTransfer(recepient, ammount) {
-  const { body } = await request('http://localhost:3000/bank-transfer', 
+  const { body } = await request('http://localhost:3000/bank-transfer',
```
Comment (austinkelleher) [inline docs/best-practices/mocking-request.md:12]: According to the `undici` [`.editorconfig`](https://github.com/nodejs/undici/blob/main/.editorconfig#L9), trimming trailing whitespace is the preferred style.

File: docs/best-practices/mocking-request.md [hunk 3]
```diff
@@ -5,17 +5,20 @@ Undici have its own mocking [utility](../api/MockAgent.md). It allow us to inter
 Example:
 
 ```js
-// index.mjs
+// bank.mjs
 import { request } from 'undici'
 
-export async function bankTransfer(recepient, ammount) {
-  const { body } = await request('http://localhost:3000/bank-transfer', 
+export async function bankTransfer(recepient, amount) {
```
Comment (austinkelleher) [inline docs/best-practices/mocking-request.md:11]: Typo correction

File: docs/best-practices/mocking-request.md [hunk 4]
```diff
@@ -5,17 +5,20 @@ Undici have its own mocking [utility](../api/MockAgent.md). It allow us to inter
...
+  const { body } = await request('http://localhost:3000/bank-transfer',
     {
       method: 'POST',
       headers: {
         'X-TOKEN-SECRET': 'SuperSecretToken',
       },
-      body: JSON.stringify({ recepient })
+      body: JSON.stringify({
+        recepient,
+        amount
```
Comment (austinkelleher) [inline docs/best-practices/mocking-request.md:20]: This was the real fix. `amount` wasn't being included in the `body` and caused the first assertion to fail.

File: docs/best-practices/mocking-request.md [hunk 5]
```diff
@@ -28,7 +31,7 @@ And this is what the test file looks like:
 // index.test.mjs
 import { strict as assert } from 'assert'
 import { MockAgent, setGlobalDispatcher, } from 'undici'
-import { bankTransfer } from './undici.mjs'
+import { bankTransfer } from './bank.mjs'
```
Comment (austinkelleher) [inline docs/best-practices/mocking-request.md:34]: See: https://github.com/nodejs/undici/pull/1341/files#r851645826

**Review by mcollina — APPROVED** [review]
lgtm all contributing to docs in this area are really welcomed!

**Pull request merged**